src/spec.c

```diff
@@ -1153,7 +1153,14 @@ void IndexSpec_FreeSync(IndexSpec *spec) {
 //---------------------------------------------------------------------------------------------
 
 void Indexes_Free(dict *d) {
-  arrayof(IndexSpec *) specs = array_new(IndexSpec *, 10);
+  // free the schema dictionary this way avoid iterating over it for each combination of
+  // spec<-->prefix
+  SchemaPrefixes_Free(ScemaPrefixes_g);
```

Comment:
This will break with swapdb and disk-less replication no?

Reply:
Swapped pointers as agrred